injected/src/features/navigator-interface.js

```diff
@@ -28,6 +29,15 @@ export default class NavigatorInterface extends ContentFeature {
                     isDuckDuckGo() {
                         return DDGPromise.resolve(true);
                     },
+                    /**
+                     * @import { MessagingInterface } from "./message-bridge/schema.js"
+                     * @param {string} featureName
+                     * @return {MessagingInterface}
+                     * @throws {Error}
+                     */
+                    createMessageBridge(featureName) {
+                        return createPageWorldBridge(featureName, args.messageSecret);
```

Comment:
this `messageSecret` matches what android call theres - in the macOS PR I've added it (with the lifespan of a tab)